service/src/main/java/com/codeforcommunity/processor/ProtectedSiteProcessorImpl.java

```diff
@@ -588,6 +594,14 @@ public List<FilterSitesResponse> filterSites(
     if (filterSitesRequest.getNeighborhoodIds() != null)
       filterCondition =
           filterCondition.and(SITES.NEIGHBORHOOD_ID.in(filterSitesRequest.getNeighborhoodIds()));
+      filterCondition = filterCondition.and(SITES.NEIGHBORHOOD_ID.in(filterSitesRequest.getNeighborhoodIds()));
```

Comment:
🔧 Is this line a duplicate of lines 595 and 596?

Reply:
I thought I made a commit to remove that line 🤦